packages/components/src/components/form/Select/Select.tsx

```diff
@@ -170,7 +170,7 @@ const Wrapper = styled.div<WrapperProps>`
         flex-wrap: nowrap;
         min-width: ${({ $minValueWidth }) => $minValueWidth};
         justify-content: ${({ $isClean }) => ($isClean ? 'flex-end' : 'flex-start')};
-        padding: 0;
+        padding: 0 25px 0 0;
```

Comment:
We try to user constants for spacing everywhere:  `${spacings.xl}` is `24px` can we use this here? 

If not and 25 is just needed in this case, no problem to use it.